Remove unused dependencies

timgl opened the pull request:
## Changes

Remove some unused dependencies (and 500 lines of yarn.lock!)

## Checklist

- [ ] All querysets/queries filter by Organization, Team, and User (if this PR affects ANY querysets/queries).
- [ ] Django backend tests (if this PR affects the backend).
- [ ] Cypress end-to-end tests (if this PR affects the frontend).

timgl committed:
Remove unused dependencies

timgl committed:
Remove more deps

mariusandra reviewed (changes requested):
Hey,
- `reselect` is a peerDependency for `kea`
- `cypress-terminal-report` is required in `cypress/{plugins,support}/index.js`
- I'm not sure of `nodemon`. How should `bin/tests` find it. Is it installed globally somehow before running tests?

timgl committed:
add reselect back in

timgl committed:
Add cypress terminal back in

timgl committed:
install cypress terminal on the fly

timgl commented:
- Added reselect back in, good catch
- Added that dep in the e2e job itself and in the installer on the fly (doesn't add much time)
- I had already added a warning that you have to install nodemon globally if you don't have it.

timgl committed:
combine
